### PR TITLE
MISUV-5335 TxM event - CL218 (R8b) -'Forecast tax calculation ' page

### PR DESCRIPTION
**Pull request by edward-dennis-ey**

### CODE REVIEW

File: it/controllers/ForecastTaxCalcSummaryControllerISpec.scala [hunk 1]
```diff
@@ -1,18 +1,75 @@
...
+import play.api.test.FakeRequest
+import testConstants.BaseIntegrationTestConstants.{testMtditid, testNino, testYear}
+import testConstants.NewCalcBreakdownItTestConstants.liabilityCalculationModelSuccessful
+import uk.gov.hmrc.auth.core.AffinityGroup.Individual
+
+object ForecastTaxSummaryControllerTestConstants {
+  val mtdItUser: MtdItUserWithNino[_] = MtdItUserWithNino(testMtditid, testNino, None, None, Some("1234567890"),
+    Some("12345-credId"), Some(Individual), None)(FakeRequest())
+
+  val taxableIncome = 12500
```
Comment (avelx) [inline it/controllers/ForecastTaxCalcSummaryControllerISpec.scala:20]: I would assume this should be an amount which should be BigDecimal ?

File: app/audit/models/ForecastTaxCalculationAuditModel.scala [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import utils.Utilities._
+
+
+case class ForecastTaxCalculationAuditModel(user: MtdItUserWithNino[_], endOfYearEstimate: EndOfYearEstimate)
+  extends ExtendedAuditModel with ImplicitDateParser {
+
+  override val transactionName: String = enums.TransactionName.ForecastTaxCalculation
+  override val auditType: String = enums.AuditType.ForecastTaxCalculation
+
+  private val totalEstimatedIncome: Option[Int] = endOfYearEstimate.totalEstimatedIncome
```
Comment (avelx) [inline app/audit/models/ForecastTaxCalculationAuditModel.scala:33]: not sure if we should use Option[Int] here to represent amount

File: it/controllers/ForecastTaxCalcSummaryControllerISpec.scala (hunk 1 above)
Comment (edward-dennis-ey) [reply it/controllers/ForecastTaxCalcSummaryControllerISpec.scala:20]: This is an Int in the IncomeSource case class so I think this is okay?

File: app/audit/models/ForecastTaxCalculationAuditModel.scala (hunk 2 above)
Comment (avelx) [reply app/audit/models/ForecastTaxCalculationAuditModel.scala:33]: clarification required from the BAs level regarding format for each amount we store in this event; I've asked @Olga to clarify requirements for this story;

**Review by avelx — APPROVED** [review]
approved

**Pull request merged**